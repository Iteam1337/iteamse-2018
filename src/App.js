// @flow

import 'what-input'
import * as React from 'react'
import Route from 'react-router-dom/Route'
import Switch from 'react-router-dom/Switch'
import { injectGlobal } from 'styled-components'

import Home from './pages/Home'
import About from './pages/About'
import HowWeWork from './pages/HowWeWork'
import Team from './pages/Team'
import Cases from './pages/Cases'
import Case from './pages/Case'
import Work from './pages/Work'
import OpenPosition from './pages/OpenPosition'
import Ops from './pages/Ops'
import Footer from './components/Footer/Footer'
import MobileMenu from './components/MobileMenu/MobileMenu'

injectGlobal`
  body {
    -webkit-font-smoothing: antialiased;
    font-family: 'Roboto', sans-serif;
    font-weight: 300;
    margin: 0;
    padding: 0;
  }
`

const App = () => (
  <React.Fragment>
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/om-oss" component={About} />
      <Route path="/hur-vi-jobbar" component={HowWeWork} />
      <Route path="/teamet" component={Team} />
      <Route exact path="/case" component={Cases} />
      <Route path="/case/:slug" component={Case} />
      <Route exact path="/jobba-hos-oss" component={Work} />
      <Route path="/jobba-hos-oss/:id" component={OpenPosition} />
      <Route path="/ops" component={Ops} />
    </Switch>
    <MobileMenu />
    <Footer />
  </React.Fragment>
)

export default App
