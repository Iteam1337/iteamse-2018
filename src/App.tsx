import * as React from 'react'
import { Route, Switch } from 'react-router-dom'
import { createGlobalStyle } from './theme'

import Footer from './components/Footer/Footer'
import MobileMenu from './components/MobileMenu/MobileMenu'
import About from './pages/About'
import Case from './pages/Case'
import Cases from './pages/Cases'
import Home from './pages/Home'
import HowWeWork from './pages/HowWeWork'
import Offers from './pages/Offers'
import OpenPosition from './pages/OpenPosition'
import Ops from './pages/Ops'
import Team from './pages/Team'
import TeamMember from './pages/TeamMember'
import Work from './pages/Work'

const GlobalStyle = createGlobalStyle`
  body {
    -webkit-font-smoothing: antialiased;
    font-family: 'Roboto', sans-serif;
    font-weight: 300;
    margin: 0;
    padding: 0;
  }
`

const App = () => (
  <>
    <GlobalStyle />
    <Switch>
      <Route exact path="/" component={Home} />
      <Route path="/om-oss" component={About} />
      <Route path="/hur-vi-jobbar" component={HowWeWork} />
      <Route exact path="/medarbetare" component={Team} />
      <Route path="/medarbetare/:shortName" component={TeamMember} />
      <Route exact path="/case" component={Cases} />
      <Route path="/case/:slug" component={Case} />
      <Route exact path="/jobba-hos-oss" component={Work} />
      <Route path="/jobba-hos-oss/:id" component={OpenPosition} />
      <Route path="/ops" component={Ops} />
      <Route path="/erbjudanden" component={Offers} />
    </Switch>
    <MobileMenu />
    <Footer />
  </>
)

export default App
