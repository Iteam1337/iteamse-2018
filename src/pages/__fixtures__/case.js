export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
