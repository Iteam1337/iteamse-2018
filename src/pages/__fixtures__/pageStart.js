export const pageStart = {
  __typename: 'PageStart',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/6WZT7iYMZGkuEUIc4S0KgK/499955f736b51dfde591a5195803a445/start_optimized.jpg',
  headerText1: 'Vi digitaliserar företag och organisationer',
  headerText2: 'genom strategi, kod och kultur',
  headerTextBgColor: '#FF3B5C',
  codeMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2mxfqVCqiw4CQYguMSYwYU/b9494a7ffcc913a275984ba7db02a1bf/dator.png',
  codeText:
    'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.',
  codeTitle: 'Kod',
  codeLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png',
  codeSlug: 'arbetsformedlingen',
  codeLinkText: 'Läs om hur vi hjälper Arbetsförmedlingen med kod',
  cultureMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2y9WBlATsssIcSQmuQ6Ky8/32e27eb88cb9d3063be20803442e3a02/chat.png',
  cultureText:
    'Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.',
  cultureTitle: 'Kultur',
  cultureLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png',
  cultureSlug: 'seb',
  cultureLinkText: 'Läs om hur vi hjälper SEB med kultur',
  strategyMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/d34548ee301d4d3f315e2ee313089093/Iteam-03-e.png',
  strategyText:
    'Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.',
  strategyTitle: 'Strategi',
  strategyLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png',
  strategySlug: 'tsab',
  strategyLinkText: 'Läs om hur vi hjälper Taxi Stockholm med strategi',
}
