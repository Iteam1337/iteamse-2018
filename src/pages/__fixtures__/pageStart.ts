export const pageStart = {
  __typename: 'PageStart',
  codeLinkText: 'Läs om hur vi hjälper Arbetsförmedlingen med kod',
  codeLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png',
  codeMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2mxfqVCqiw4CQYguMSYwYU/b9494a7ffcc913a275984ba7db02a1bf/dator.png',
  codeSlug: 'arbetsformedlingen',
  codeText:
    'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.',
  codeTitle: 'Kod',
  contactTitle: 'Hör av dig till oss!',
  cultureLinkText: 'Läs om hur vi hjälper SEB med kultur',
  cultureLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png',
  cultureMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/2y9WBlATsssIcSQmuQ6Ky8/32e27eb88cb9d3063be20803442e3a02/chat.png',
  cultureSlug: 'seb',
  cultureText:
    'Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.',
  cultureTitle: 'Kultur',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/6WZT7iYMZGkuEUIc4S0KgK/499955f736b51dfde591a5195803a445/start_optimized.jpg',
  headerText1: 'Vi digitaliserar företag och organisationer',
  headerText2: 'genom strategi, kod och kultur',
  headerTextBgColor: '#FF3B5C',
  strategyLinkText: 'Läs om hur vi hjälper Taxi Stockholm med strategi',
  strategyLogo:
    '//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png',
  strategyMobileImage:
    '//images.ctfassets.net/rj4r6yfcesw5/1oqMfeZBnuQAAGweIGCKEW/d34548ee301d4d3f315e2ee313089093/Iteam-03-e.png',
  strategySlug: 'tsab',
  strategyText:
    'Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.',
  strategyTitle: 'Strategi',
  team: ['rln', 'msr'],
}
