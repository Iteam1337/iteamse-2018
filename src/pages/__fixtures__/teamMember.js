export const teamMember = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'New York',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    __typename: 'TeamMember',
  },
]
