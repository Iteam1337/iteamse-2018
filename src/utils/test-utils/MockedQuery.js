import * as React from 'react'
import { ApolloProvider } from 'react-apollo'
import ApolloClient from 'apollo-client'
import { createHttpLink } from 'apollo-link-http'
import createMockedNetworkFetch from 'apollo-mocknetworkinterface'
import { InMemoryCache } from 'apollo-cache-inmemory'
import { BrowserRouter } from 'react-router-dom'
import { ThemeProvider } from 'styled-components'
import { theme } from '../../theme'

type P = {
  children: React.Node,
  loading?: boolean,
  response: Object,
}

const MockedQuery = ({ children, loading, response = {} }: P) => {
  const createResponse = () => {
    return {
      loading: !!loading,
      data: {
        ...response,
      },
    }
  }

  const mockedNetworkFetch = createMockedNetworkFetch(createResponse, {
    latency: 0,
  })

  const client = new ApolloClient({
    link: createHttpLink({
      uri: 'http://localhost:3000',
      fetch: mockedNetworkFetch,
    }),
    cache: new InMemoryCache(),
  })

  return (
    <ApolloProvider client={client}>
      <ThemeProvider theme={theme}>
        <BrowserRouter>{children}</BrowserRouter>
      </ThemeProvider>
    </ApolloProvider>
  )
}

export default MockedQuery
