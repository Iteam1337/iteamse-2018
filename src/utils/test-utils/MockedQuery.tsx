import { InMemoryCache } from 'apollo-cache-inmemory'
import ApolloClient from 'apollo-client'
import * as React from 'react'
import { ApolloProvider } from 'react-apollo'
import { mockSingleLink } from 'react-apollo/test-utils'
import MockRouter from 'react-mock-router'
import { ThemeProvider } from 'styled-components'
import { theme } from '../../theme'

interface MockedQueryProps {
  location?: object
  mocks: any[]
  params?: object
}

const MockedQuery: React.SFC<MockedQueryProps> = ({
  children,
  location,
  mocks,
  params,
}) => {
  const link = mockSingleLink.apply(null, mocks)

  const client = new ApolloClient({
    cache: new InMemoryCache(),
    link,
  })

  return (
    <ApolloProvider client={client}>
      <ThemeProvider theme={theme}>
        <MockRouter location={location} params={params}>
          {children}
        </MockRouter>
      </ThemeProvider>
    </ApolloProvider>
  )
}

export default MockedQuery
