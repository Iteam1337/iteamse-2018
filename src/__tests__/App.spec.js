import React from 'react'
import App from '../App'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../utils/test-utils/MockedQuery'
import { pageStart } from '../pages/__fixtures__/pageStart'
import { teamMember } from '../pages/__fixtures__/teamMember'
import { addresses } from '../components/Footer/__fixtures__/addresses'

describe('App', () => {
  const mockedResponse = {
    addresses,
    pageStart,
    teamMember,
  }

  it('renders App', async () => {
    const { getByText, container } = render(
      <MockedQuery response={mockedResponse}>
        <App />
      </MockedQuery>
    )

    await wait(() => getByText('iteam'))

    expect(container).toMatchSnapshot()
  })
})
