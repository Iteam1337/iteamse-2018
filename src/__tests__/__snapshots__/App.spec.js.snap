// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App 1`] = `
<div>
  <div
    style="height: 700px;"
    class="lazyload-placeholder"
  />
  <div
    class="GridColumn__GridColumnClean-lcQAdD eeOKQI"
  >
    <div
      class="Home__Content-s15fccpq-0 htMAwU"
    >
      <img
        class="Home__StylizedIteam-s15fccpq-2 hgtCAO"
        src="iteam_stylized.png"
      />
      <div
        class="Home__Texts-s15fccpq-1 eCpoTV"
      >
        <div
          class="s3nsc5b-0-PaddedRow-hWTqUj khULXJ"
        >
          <h1
            class="H1-x5hh3k-0 eejZji"
          >
            Kod
          </h1>
          <div
            class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
          >
            <div>
              <p>
                Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.
              </p>
            </div>
          </div>
          <img
            class="Home__CompanyLogo-s15fccpq-3 bmmzFx"
            src="//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png"
          />
          <a
            class="Link__StyledLink-kqcp4a-0 exsHFb"
            href="/case/arbetsformedlingen"
          >
            Läs om hur vi hjälper Arbetsförmedlingen med kod
          </a>
        </div>
        <div
          class="s3nsc5b-0-PaddedRow-hWTqUj khULXJ"
        >
          <h1
            class="H1-x5hh3k-0 eejZji"
          >
            Strategi
          </h1>
          <div
            class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
          >
            <div>
              <p>
                Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.
              </p>
            </div>
          </div>
          <img
            class="Home__CompanyLogo-s15fccpq-3 bmmzFx"
            src="//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png"
          />
          <a
            class="Link__StyledLink-kqcp4a-0 exsHFb"
            href="/case/tsab"
          >
            Läs om hur vi hjälper Taxi Stockholm med strategi
          </a>
        </div>
        <div
          class="s3nsc5b-0-PaddedRow-hWTqUj khULXJ"
        >
          <h1
            class="H1-x5hh3k-0 eejZji"
          >
            Kultur
          </h1>
          <div
            class="Paragraph__StyledParagraph-s13mwy58-0 kBKPHf"
          >
            <div>
              <p>
                Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.
              </p>
            </div>
          </div>
          <img
            class="Home__CompanyLogo-s15fccpq-3 bmmzFx"
            src="//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png"
          />
          <a
            class="Link__StyledLink-kqcp4a-0 exsHFb"
            href="/case/seb"
          >
            Läs om hur vi hjälper SEB med kultur
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD lbDVJp"
  >
    <div
      class="OperationsBanner__Overlay-s6plh63-0 eUEmcx"
    />
    <div
      class="OperationsBanner__Content-s6plh63-1 YGXrX"
    >
      <h2
        class="H2-s1x2w9v8-0 exLhgN"
      >
        Visste du att vi även erbjuder drift & support?
      </h2>
      <a
        class="Button-s1kb8b7s-0 bphzbK"
        href="/ops"
      >
        Iteam Operations
      </a>
    </div>
  </div>
  <div
    class="s7kn9a1-0-GridColumn__GridColumnClean-lcQAdD dXaaGv"
  >
    <div
      class="TeamMembers__Members-gavpkf-0 kvZgxv"
    >
      <div
        class="Colleague-s1m0ml37-0 cknafA"
      >
        <div
          class="TeamMember__Title-ahjagd-0 elIZCP"
        >
          Cookie master
        </div>
        <div
          class="TeamMember__Location-ahjagd-1 dnfurN"
          data-test="team-member-location"
        >
          Stockholm
        </div>
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cmm"
        >
          <div
            class="TeamMember__AvatarWrap-ahjagd-2 hOPutZ"
          >
            <div
              style="height: 226px;"
              class="lazyload-placeholder"
            />
          </div>
        </a>
        <div
          class="TeamMember__Title-ahjagd-0 elIZCP"
        >
          Cookie M
        </div>
        <a
          class="NativeLink-s1qo6rwq-0 jNinQt"
          href="mailto:cm@sesame.com"
        >
          cm@sesame.com
        </a>
      </div>
      <div
        class="Colleague-s1m0ml37-0 cknafA"
      >
        <div
          class="TeamMember__Title-ahjagd-0 elIZCP"
        >
          1 ahahah
        </div>
        <div
          class="TeamMember__Location-ahjagd-1 dnfurN"
          data-test="team-member-location"
        >
          Göteborg
        </div>
        <a
          class="PrefetchLink__StyledLink-n3ecxs-0 JPiJu"
          href="/teamet/cvc"
        >
          <div
            class="TeamMember__AvatarWrap-ahjagd-2 hOPutZ"
          >
            <div
              style="height: 226px;"
              class="lazyload-placeholder"
            />
          </div>
        </a>
        <div
          class="TeamMember__Title-ahjagd-0 elIZCP"
        >
          Count van Count
        </div>
        <a
          class="NativeLink-s1qo6rwq-0 jNinQt"
          href="mailto:count@sesame.com"
        >
          count@sesame.com
        </a>
      </div>
    </div>
  </div>
  <div
    class="MobileMenu__Wrap-s128ahbw-0 eHbiAP"
  >
    <button
      class="MenuButton__Button-cyy49g-0 xzjBp"
      aria-label="Menu button"
      data-testid="btn-mobile-menu"
      role="button"
    >
      <div
        class="MenuButton__Hamburger-cyy49g-2 ivvWAI"
      >
        <div
          class="MenuButton__Bar-cyy49g-1 dymejY"
        />
        <div
          class="MenuButton__Bar-cyy49g-1 dymejY"
        />
        <div
          class="MenuButton__Bar-cyy49g-1 dymejY"
        />
      </div>
    </button>
  </div>
  <div
    class="GridColumn__GridColumnClean-lcQAdD cMYMfO"
  >
    <div
      class="Footer__FooterSections-xtla5m-0 cgXbGm"
    >
      <div
        class="Location__Wrap-s1et3i43-0 gDiDVe"
      >
        <div
          class="Location__Title-s1et3i43-1 hOnjvR"
        >
          Iteam Göteborg
        </div>
        <div
          class="Location__Address-s1et3i43-2 bVugku"
        >
          <ul
            class="UnstyledList-s1ntpurm-0 gEDRFE"
          >
            <li>
              Haga Kyrkogata 28
            </li>
            <li>
              411 23
              Göteborg
            </li>
          </ul>
        </div>
        <ul
          class="UnstyledList-s1ntpurm-0 gEDRFE"
        >
          <li>
            +46 734 029 112
          </li>
          <li>
            johanna.grahn@iteam.se
          </li>
          <li>
            ORG: 
            556551-6928
          </li>
        </ul>
      </div>
      <div
        class="Location__Wrap-s1et3i43-0 gDiDVe"
      >
        <div
          class="Location__Title-s1et3i43-1 hOnjvR"
        >
          Iteam Stockholm
        </div>
        <div
          class="Location__Address-s1et3i43-2 bVugku"
        >
          <ul
            class="UnstyledList-s1ntpurm-0 gEDRFE"
          >
            <li>
              Östermalmsgatan 26A
            </li>
            <li>
              114 26
              Stockholm
            </li>
          </ul>
        </div>
        <ul
          class="UnstyledList-s1ntpurm-0 gEDRFE"
        >
          <li>
            +46 826 70 90
          </li>
          <li>
            info@iteam.se
          </li>
          <li>
            ORG: 
            556551-6928
          </li>
        </ul>
      </div>
      <div
        class="Footer__SocialMedia-xtla5m-2 fltlKn"
      >
        <div
          class="Footer__Title-xtla5m-1 hAElLH"
        >
          Följ oss
        </div>
        <ul
          class="UnstyledList-s1ntpurm-0 gEDRFE"
        >
          <li
            class="Footer__SocialMediaItem-xtla5m-3 eUeiui"
          >
            <img
              class="Footer__SocialMediaIcon-xtla5m-4 bRvQuN"
              src="icon_fb.svg"
              alt=""
            />
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="https://www.facebook.com/iteam1337/"
            >
              Facebook
            </a>
          </li>
          <li
            class="Footer__SocialMediaItem-xtla5m-3 eUeiui"
          >
            <img
              class="Footer__SocialMediaIcon-xtla5m-4 bRvQuN"
              src="icon_instagram.svg"
              alt=""
            />
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="https://instagram.com/iteam1337"
            >
              Instagram
            </a>
          </li>
          <li
            class="Footer__SocialMediaItem-xtla5m-3 eUeiui"
          >
            <img
              class="Footer__SocialMediaIcon-xtla5m-4 bRvQuN"
              src="icon_linkedin.svg"
              alt=""
            />
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="https://www.linkedin.com/company/270569/"
            >
              LinkedIn
            </a>
          </li>
          <li
            class="Footer__SocialMediaItem-xtla5m-3 eUeiui"
          >
            <img
              class="Footer__SocialMediaIcon-xtla5m-4 bRvQuN"
              src="icon_youtube.svg"
              alt=""
            />
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="https://www.youtube.com/channel/UCU2TdLZ7p0jEuuGUxaod4lw"
            >
              YouTube
            </a>
          </li>
          <li
            class="Footer__SocialMediaItem-xtla5m-3 eUeiui"
          >
            <img
              class="Footer__SocialMediaIcon-xtla5m-4 bRvQuN"
              src="icon_mynewsdesk.svg"
              alt=""
            />
            <a
              class="NativeLink-s1qo6rwq-0 jNinQt"
              href="http://www.mynewsdesk.com/se/iteam"
            >
              MyNewsDesk
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
